app/uk/gov/hmrc/cataloguefrontend/config/CatalogueConfig.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cataloguefrontend.config
+
+import com.google.inject.{Inject, Singleton}
+import play.api.Configuration
+
+@Singleton
+class CatalogueConfig @Inject()(configuration: Configuration) {
+  val shutterGroup    = configuration.get[String]("perms.shutter.group")
+  val shutterAnyGroup = configuration.get[String]("perms.shutter-any.group")
```

Comment:
Just wondering if `shutter-any` indicates clearly enough the god mode status of that group?